src/Fantomas/CodePrinter.fs

```diff
@@ -2395,7 +2395,10 @@ and genType astContext outerBracket t =
 
             fun (ctx:Context) ->
                 isShortExpression ctx.Config.MaxRecordWidth shortExpression longExpression ctx
-
+        | TParen(innerT) ->
+            sepOpenT +>
```

Comment:
in this case, to print a `TParen`, we render the parens around the rendering of the inner SynType.